packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -216,18 +216,35 @@ class FuchsiaDevice extends Device {
 
   /// List the ports currently running a dart observatory.
   Future<List<int>> servicePorts() async {
-    final String lsOutput = await shell('ls /tmp/dart.services');
-    return parseFuchsiaDartPortOutput(lsOutput);
+    final String findOutput = await shell('find /hub -name vmservice-port');
+    if (findOutput.trim() == '') {
+      throwToolExit('No Dart Observatories found. Are you running a debug build?');
+      return null;
+    }
+    final List<int> ports = <int>[];
+    for (var path in findOutput.split('\n')) {
```

Comment:
nit: I think the flutter analyze rules require types (see: the other for-loops)

Reply:
Thanks! Done.